File: trunk/RRComSSys/Projects/CVM/RRComSSys.CVM/src/edu/fiu/cis/rrcomssys/cvm/mte/CMLDocumentException.java
package edu.fiu.cis.rrcomssys.cvm.mte;

public class CMLDocumentException extends Exception
{
	/* Constants */
	private static final long serialVersionUID = -8804326482040251823L;

	/* Constructors & Initializers */
	public CMLDocumentException()
	{
		super();
	}

	public CMLDocumentException(String message, Throwable cause)
	{
		super(message, cause);
	}

	public CMLDocumentException(String message)
	{
		super(message);
	}

	public CMLDocumentException(Throwable cause)
	{
		super(cause);
	}
}

File: trunk/RRComSSys/Projects/CVM/RRComSSys.CVM.ObjectModel/CMLDocumentException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;

namespace RRComSSys.CVM.ObjectModel
{
	public class CMLDocumentException : Exception
	{
		#region Constructors & Initializers
		public CMLDocumentException()
			: base() { }

		public CMLDocumentException(String message)
			: base(message) { }

		public CMLDocumentException(String message, Exception innerException)
			: base(message, innerException) { }

		public CMLDocumentException(SerializationInfo info, StreamingContext context)
			: base(info, context) { }
		#endregion
	}
}
